pkg/apis/storageos/v1/storageoscluster_types.go

```diff
@@ -150,6 +150,22 @@ type StorageOSClusterSpec struct {
 	// Disable Telemetry.
 	DisableTelemetry bool `json:"disableTelemetry"`
 
+	// Disable TCMU can be set to true to disable the TCMU storage driver.  This
+	// is recommended when there are multiple storage systems running on the
```

Comment:
```suggestion
	// is necessary/required when there are multiple storage systems running on the
```

Again I'd prefer to see absolute language here.

Reply:
done